conformance/echo-basic/echo-basic.go

```diff
@@ -267,7 +292,7 @@ func processError(w http.ResponseWriter, err error, code int) { //nolint:unparam
 	_, _ = w.Write(body)
 }
 
-func listenAndServeTLS(addr string, serverCert string, serverPrivKey string, clientCA string, handler http.Handler) error {
+func listenAndServeTLS(addr string, serverCert string, serverPrivKey string, clientCA string, clientCert string, clientPrivKey string, handler http.Handler) error {
```

Comment:
I don't think `clientCert` and `clientPrivKey` is needed by the backend service to validate the gateway.

IIUC the backend server can use the `clientCA` to validate the gateway's client cert.